src/man/sss-certmap.5.xml

```diff
@@ -52,6 +52,14 @@
             Internally the priority is treated as unsigned 32bit integer, using
             a priority value larger than 4294967295 will cause an error.
         </para>
+        <para>
+            If multiple rules have the same priority and only one of the related
+            matching rules applies this rule will be chosen. If there are
```

Comment:
I think you need a comma after rules applies in this line.  I think it should be:

`matching rules applies, this rule will be chosen`